samples/subsys/mgmt/mcumgr/smp_svr/README.rst

```diff
@@ -47,6 +47,14 @@ Bluetooth Low Energy (BLE) and do not have a built-in or pluggable BLE radio,
 you can build one and use it following the instructions in
 :ref:`bluetooth-hci-uart-bluez`.
 
+Building and Running using NCS
+******************************
+Building the sammple as a part off nRF Connect SDK is simplified in comparasion
+to the vanilla zephyr sample.
+
+.. include:: /includes/build_and_run.txt
```

Comment:
Is this file missing, or is it in the nrf repo?

Reply:
This file is in the nrf repository, so this won't work unfortunately, and you'll need to include the text manually. :(
We also don't have automatic linking set up this way around (you can link from the NCS docs to the Zephyr docs, but not from Zephyr to NCS), so the only way to link will be to use an absolute link ( https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_programming.html#gs-programming ) - which will point to master and not to a specific release, and it'll point to the online doc even when users build the docs locally. I think we can live with those problems though.